src/engine/IdTable.h

```diff
@@ -636,18 +650,18 @@ class IdTableImpl<0, DATA> {
   size_t _cols = 0;
 
  public:
-  IdTableImpl() = default;
+  IdTableImpl(Alloc al) : _data{al} {};
 
  protected:
   template <int INCOLS, typename INDATA>
-  explicit IdTableImpl(const IdTableImpl<INCOLS, INDATA>& o)
+  explicit IdTableImpl(const IdTableImpl<INCOLS, INDATA, Alloc>& o)
       : _size(o._size),
         _capacity(o._capacity),
         _data(o._data),
```

Comment:
Old code: Why no std::move(o._data) here, like in other places in the code?

Reply:
In this place it would be wrong, since it is a copy constructor. (It wouldn't even compile because of the `const`